docs/_static/jquery.js

```diff
@@ -1,2 +1,4 @@
\ No newline at end of file
+/*! jQuery v3.2.1 | (c) JS Foundation and other contributors | jquery.org/license */
```

Comment:
You need to manually swap this with the v3.5.0 that was overwritten. This is unfortunately something we have to do until Sphinx updates their version dependency, unless we aren't concerned about the security warning

Reply:
Good point, but let me come back to this after this merge --to fix security warning, IF it shows up. I am using the same setup for Seq2Pat and it does not give a warning --so this might just work

Reply:
Gotch you, that make sense. Let me go through with this PR and we can fix the security warning separately.